packetbeat/_meta/beat.reference.yml

```diff
@@ -316,6 +316,26 @@ packetbeat.protocols:
   # incoming responses, but sent to Elasticsearch immediately.
   #transaction_timeout: 10s
 
+- type: smtp
+  # SMTP port for traffic monitoring.
+  ports: [25]
+
+  # If this option is enabled, the SMTP DATA request (payload) headers are
+  # included in published events. The default is false.
+  #send_data_headers: false
+
+  # If this option is enabled, the raw message of SMTP DATA request (payload)
+  # body is included in published events. The default is false.
+  #send_data_body: false
+
+  # If this option is enabled, the raw message of the request (`request` field)
+  # is sent to Elasticsearch. The default is false.
+  #send_request: false
+
+  # If this option is enabled, the raw message of the response (`response`
+  # field) is sent to Elasticsearch. The default is false.
+  #send_response: false
```

Comment:
From description it's somewhat unclear to me how these settings are related to each other. If `send_request` is enabled, does this mean `send_data_headers` and `send_data_body` are automatically enabled as well?

Reply:
> If send_request is enabled, does this mean send_data_headers and send_data_body are automatically enabled as well?

No, the 2 pairs are not related. As with HTTP, the user enables `send_request` and `send_response` if they want the raw data included under the event's top-level `request` and `response` fields.

`send_data_headers` and `send_data_body`, on the other hand, add relevant fields to `smtp -> request` for `EOD`-type requests.

I'll try to make it clear in the description.